lib/collection/request.js

```diff
@@ -50,9 +51,11 @@ _.inherit((
             headers: new HeaderList(this, options && options.header),
 
             /**
+             * Although a similar check is being done in the .update call below, this handles falsy options as well.
```

Comment:
Could you move this as a single line comment out of the jsDoc. This ended up in the public documentation.

<img width="705" alt="screen shot 2017-08-24 at 7 41 01 pm" src="https://user-images.githubusercontent.com/8401064/29670435-3e7316ee-8904-11e7-81b2-b0187f994ff7.png">